Makefile

```
@@ -38,12 +38,16 @@ CGOFLAG ?= CGO_ENABLED=1
 # should be an absolute directory as it is used by e/Makefile too, from the e/ directory.
 RELEASE_DIR := $(CURDIR)/$(BUILDDIR)/artifacts
 
+LDFLAGS ?= -w -s $(KUBECTL_SETVERSION)
+
+# Appending new conditional settings for community build type
+LDFLAGS += $(if $(filter $(TELEPORT_BUILD_TYPE),community),-X github.com/gravitational/teleport/lib/modules.teleportBuildType=community)
```

IMO it'd be better to pull in the build type via a makefile variable, like this:
```suggestion
BINARY_LICENSE_TYPE ?= community
LDFLAGS ?= -w -s $(KUBECTL_SETVERSION) -X github.com/gravitational/teleport/lib/modules.teleportBuildType=$(BINARY_LICENSE_TYPE)
```

unless there is a business reason to only allow overwriting this with "community" by default.

---

For now, we only want the value to be community or nothing